### PR TITLE
Integrate HMTK with Travis CI

### LINKED ISSUE #73: Missing setup.py
A `setup.py` must be added to allow easy installation of the toolkit without the need to change the `PYTHONPATH`, like:

``` bash
pip install -e hmtk/
```

### PR DESCRIPTION
**Pull request by daniviga**
- @micheles fixed most of the tests
- a new set of `requirements-*.txt` has been added (like we already do in oq-engine/hazardlib):
  - `requirements-dev.txt` contains the platform agnostic version of the requirements (to be used i.e. on macOS)
  - `requirements-py[27,35]-linux64.txt` contain the pre-built binary wheels for linux64, available on our [wheels mirror](http://ftp.openquake.org/python-new-wheels/) (the wheel is generated by https://github.com/gem/oq-containers/tree/master/wheels), which are used by Travis

The `setup.py` has been added too, closes #73

### CODE REVIEW

**Comment by daniviga** [comment]
The `setup.py` has been added, closes #73

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+README = """
+The hmtk is a suite of tools developed by Scientists working at the GEM (i.e.
+Global Earthquake Model) Model Facility. The main purpouse of the hmtk is to
+provide a suite of tools for the creation of PSHA input models.
+Copyright (C) 2010-2016 GEM Foundation
+"""
+
+setup(
+    name='hmtk',
+    version='0.0.1',
```
Comment (daniviga) [inline setup.py]: @mmpagani we should provide a better number here

**Comment by mmpagani** [comment]
LGTM

File: setup.py (hunk 1 above)
Comment (mmpagani) [inline setup.py]: 1.0.0

**Pull request merged**